pkg/controller/populators/import-populator.go

```diff
@@ -162,11 +162,13 @@ func (r *ImportPopulatorReconciler) reconcileTargetPVC(pvc, pvcPrime *corev1.Per
 		}
 	}
 
-	err = r.updatePVCWithPVCPrimeAnnotations(pvcCopy, pvcPrime, r.updateImportAnnotations)
-	if err != nil {
+	if pvcCopy, err = r.updatePVCWithPVCPrimeAnnotations(pvcCopy, pvcPrime, r.updateImportAnnotations); err != nil {
 		return reconcile.Result{}, err
 	}
 	if cc.IsPVCComplete(pvcPrime) && !cc.IsMultiStageImportInProgress(pvc) {
+		if err = r.updatePVCWithPVCPrimeLabels(pvcCopy, pvcPrime.GetLabels()); err != nil {
```

Comment:
As we discussed offline we probably have a bigger issue here in that we'd never make our way back to this part on a requeue, since the PVC may already be succeeded.
I think we can let the loop through if labels aren't there for now and fix properly later

Reply:
Looks to be fixed by https://github.com/kubevirt/containerized-data-importer/pull/3103/commits/c24717c44ac17f436eb9dbd5886afaf3106563ea